src/ui/css/styles.scss

```diff
@@ -1,2 +1,12 @@
 @import '~core/css/inc/mixins';
 @import './vars';
+
+.Card-header,
+.Definition-dd,
+.MetadataCard-list,
+.ShowMoreCard-contents,
+.UserCollection-name,
+.UserReview,
+.UserReview-body.ShowMoreCard {
+  overflow-wrap: break-word;
```

Comment:
Do we really have to declare each css class?

Reply:
I think so, do you have any ideas I could look into?

Reply:
I pushed an update, could you please take a look and test locally if that would work?